statsmodels/stats/pscore_match.py

```diff
@@ -0,0 +1,242 @@
+'''Propensity Score Matchig
+
+
+Author: 
+License: BSD-3
+
+'''
+
+import statsmodels.api as sm
+import numpy as np
+from statsmodels.stats.weightstats import CompareMeans, DescrStatsW
+import pandas as pd
+from scipy import stats
+
+
+class PropensityScoreMatch(object):
+    def __init__(self, assigment_index, covariates, treatment_objective_variable,\
```

Comment:
It would be nice to have the option of passing a formula and dataframe for the propensity score model, or a numpy exog matrix as here ('covariates' should probably be `exog` for consistency).